service/evidence/evidence_store.go

```diff
@@ -95,28 +95,36 @@
 
 // StoreEvidences is a method implementation of the evidenceServer interface: It receives evidences and stores them
 func (s *Service) StoreEvidences(stream evidence.EvidenceStore_StoreEvidencesServer) (err error) {
-	var req *evidence.StoreEvidenceRequest
+	var (
+		req *evidence.StoreEvidenceRequest
+		res *evidence.StoreEvidenceResponse
+	)
 
 	for {
 		req, err = stream.Recv()
 
+		// If no more input of the stream is available, return
+		if err == io.EOF {
+			return nil
+		}
 		if err != nil {
-			// If no more input of the stream is available, return SendAndClose `error`
-			if err == io.EOF {
-				log.Infof("Stopped receiving streamed evidences")
-				return stream.SendAndClose(&emptypb.Empty{})
-			}
-
-			return err
+			log.Errorf("EvidenceStore: Cannot receive stream request: %v", err)
```

Comment:
Same as for assessment: Omit EvidenceStore and consider to avoid duplicate string